app/storybook/documentation/ui-kit/components/PieChart.mdx

```diff
@@ -0,0 +1,51 @@
+import { ArgTypes, Meta } from '@storybook/blocks'
+import LinkTo from '@storybook/addon-links/react'
+import { prettier } from '../../../src/utils'
+import { SourceCode } from '../../../.storybook/blocks/SourceCode'
+import * as PieChartStories from '../../../../../packages/ui-kit/src/components/PieChart/PieChart.stories'
+import * as PieChartQueryPropsStories from '../../../../../packages/ui-kit/src/components/PieChart/PieChartQueryProps.stories'
+import * as PieChartPropsStories from '../../../../../packages/ui-kit/src/components/PieChart/PieChartProps.stories'
+
+<Meta title="Components/PieChart" of={PieChartStories} />
+
+# PieChart overview
+
+The PieChart component is designed to create pie chart (AKA doughnut chart) visualizations for your front-end applications. 
+With two props in Static mode, `headers` and `rows`, the PieChart component provides a simple way to display your data. 
+If you opt for the Connected mode, the component will accept the `query` prop, which loads the data automatically using Propel's GraphQL API.
```

Comment:
I think it's important to also mention that for now the PieChart component supports only one dimension